arkane/util.py

```diff
@@ -73,3 +77,46 @@ def determine_qm_software(fullpath):
             raise InputError(f'The file at {fullpath} could not be identified as a '
                              'Gaussian, Molpro, QChem, or TeraChem log file.')
     return software_log
+
+
+def read_supporting_information(path):
```

Comment:
since Arkane wrote the SI file, do we really need to parse it?

Reply:
But isn't this YAML file being saved before Arlane termiates? Or did I miss the expected workflow?

Reply:
The csv file is being saved before Arkane terminates, and contains information not found in the species YAML file. I added this in case the user ran an Arkane job say a month ago and then realized that they wanted to update a reference species with that information (they would only be able to use the information contained in these files, and the CSV file is the only place that has this information).

That being said, I have decided to cut this function from the PR. We would have to add back in the `util.py` file just for this, and we would have to maintain this function whenever we change the format. If the user wants to add in this information they can always do so by hand or write a script for themselves.

The preferred way going forward would be to have Arkane update reference species files before terminating, but this is will be included in a future PR.